pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -117,6 +117,9 @@ class LightningGPU(LightningQubit):
     }
 
     def __init__(self, wires, *, shots=None, sync=True):
+        if shots is not None:
```

Comment:
```suggestion
        if shots:
```

should allow `shots=0` and `None` to skip this.